src/sync/discovery.ts

```diff
@@ -51,8 +53,12 @@ export const discovery = (app: Application, queues) => async (job, logger: Logge
 			repos
 		});
 
-		// Create job
-		queues.installation.add({ installationId, jiraHost, startTime });
+		if (await booleanFlag(BooleanFlags.USE_SQS_FOR_BACKFILL, false, jiraHost)) {
```

Comment:
this FF evaluation isn't really necessary anymore, since it's also evaluated in the `backfillQueueSupplier()`, is it? It doesn't really hurt, though, I guess.

Reply:
Correct. Yeah, added just in case